substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/snippets/SubstrateGraphBuilderPlugins.java

```diff
@@ -371,6 +371,13 @@ private static void interceptProxyInterfaces(GraphBuilderContext b, ResolvedJava
              */
             List<Class<?>> classList = new ArrayList<>();
             FixedNode successor = unwrapNode(newArray.next());
+            if (successor instanceof EnsureClassInitializedNode) {
+                /*
+                 * Skip EnsureClassInitialized and Begin nodes.
```

Comment:
If this comment stays, I would make it one line.

Reply:
I think that one comment, before the if statement, is enough. So I removed this one.